src/krux/pages/home_pages/wallet_descriptor.py

```diff
@@ -84,44 +84,49 @@ def _load_wallet(self):
                 _, wallet_data = utils.load_file(
                     (DESCRIPTOR_FILE_EXTENSION, JSON_FILE_EXTENSION)
                 )
+                persisted = True
             except OSError:
                 pass
+        else:
```

Comment:
If you made the above change to initialize `persisted as False`, you can remove this `else:`

Reply:
I believe that odudex already caught this when breaking-up the loading of descriptors from in-serial (via camera, then from sdcard) to parallel choices of how to load.